app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
```

Comment:
Let's add a `dependent :destroy` here.

Reply:
Added dependant: :destroy to belongs to and has many assosiations here.